_episodes/04-quality-control-2.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Quality control"
+teaching: 20
+exercises: 0
+questions:
+- "How can we carry out basic quality control and quality assurance in spreadsheets?"
+objectives:
+- "Apply quality control techniques to identify errors in spreadsheets and limit incorrect data entry."
+keypoints:
+- "Always copy your original spreadsheet file and work with a copy so you don't affect the raw data."
+- "Use data validation to prevent accidentally entering invalid data."
+- "Use sorting to check for invalid data."
+- "Use conditional formatting (cautiously) to check for invalid data."
+---
+
+Authors:**Christie Bahlai**, **Aleksandra Pawlik**<br>
```

Comment:
including author names in the lesson is a remanent of the old DC template, we usually don't include them, so it's probably best to remove them here.

Reply:
Yes, definitely. I hadn't gotten to this episode yet. I will remove those when I do the PR for the second half of this lesson.